webui/src/lib/onboarding/repoOnboardingService.tsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable react/jsx-no-target-blank */
+import React from "react";
+import { OnBoardingSteps } from "./types";
+import { NavigateFunction } from "react-router-dom";
+import { branches, retention, branchProtectionRules, objects } from "../api";
+
+const IMPORT_BRANCH_REGEX = /^_(.*)_imported$/;
+const HOOKS_PATH = "_lakefs_actions/";
+const SAMPLE_REPO_FILE_NAME = "lakes.parquet";
```

Comment:
I _think_ that this is a path not a file name.  lakeFS doesn't files, and this manifests in various ways.  Sometimes we're lax and say "file" anyway.  But in particular, it doesn't have file names because it doesn't have directories -- so just paths.

```suggestion
const SAMPLE_REPO_OBJECT_PATH = "lakes.parquet";
```

Reply:
Done